setup.cfg

```diff
@@ -85,7 +85,7 @@ filterwarnings =
 [flake8]
 max-complexity = 15
 exclude = .ecosystem,.eggs,.git,.tox,.venv,build,dist,docs,examples,falcon/bench/nuts
-ignore = F403,W504
+extend-ignore = F403,W504,E203,I202
 max-line-length = 99
```

Comment:
Shouldn't we now decrease this to 88?

Reply:
Yes, I guess so

Reply:
Note that we have quite a bit of comment/documentation that are over 88 chars